sssom/parsers.py

```diff
@@ -307,6 +339,12 @@ def from_sssom_json(
     mapping_set = cast(
         MappingSet, JSONLoader().load(source=jsondoc, target_class=MappingSet)
     )
+    # ** Will need to remove this line
+    # ** once linkml-runtime fixes JSONLoader.load()
```

Comment:
Add the linkml issue that relates to this.

Reply:
And a # TODO tag

Reply:
good catch! It has been resolved by `inlined_as_list: true` for mappings . This is not needed anymore. Deleting.